roles/openshift_hosted/README.md

```diff
@@ -33,13 +33,17 @@ variables also control configuration behavior:
 
 | Name                                         | Default value | Description                                                                  |
 |----------------------------------------------|---------------|------------------------------------------------------------------------------|
-| openshift_hosted_registry_glusterfs_swap     | False         | Whether to swap an existing registry's storage volume for a GlusterFS volume |
-| openshift_hosted_registry_glusterfs_swapcopy | True          | If swapping, also copy the current contents of the registry volume           |
+| openshift_hosted_registry_storage_glusterfs_ips      | []            |A list of IP addresses for the GlusterFS cluster to use for hosted registry storage|
+| openshift_hosted_registry_storage_glusterfs_path     | []            |If we use external gluster we need to provide volume name from gluster cluster for mounting|
+
+**Note:** if you have host group `glusterfs_registry` set in your invetory, variable `openshift_hosted_registry_storage_glusterfs_ips`, CAN'T be used because this is 2 different use-cases - CNS/CRS vs External Native Gluster. 
```

Comment:
Use the wording we already have elsewhere:

```
**Note:** Configuring a value for 
openshift_hosted_registry_storage_glusterfs_ips and with a glusterfs_registry 
host group is not allowed. Specifying a glusterfs_registry host group 
indicates that a new GlusterFS cluster should be configured, whereas 
specifying openshift_hosted_registry_storage_glusterfs_ips indicates wanting 
to use a pre-configured GlusterFS cluster for the registry storage.
```
I don't want product names in this documentation. :)